10.9-libcxx/stable/main/finkinfo/devel/pylint-py.info

```diff
@@ -68,5 +76,14 @@ customizable, and you can easily write a small plugin to add a
 personal feature.
 << 
 
-# Info2
+DescPackaging: <<
+Added Sphinx build of HTML docs, changed names of versioned executables
+to standard -py%type_pkg[python] scheme (pylint-gui no longer included).
+2to3 treatment of sources no longer required.
+Added cleanup of byte-compiled files to InstallScript.
+3 test failures on Python 3.x, > 222 (basically the entire test_stdlib)
+mostly unicode-related on 2.7.14.
+<<
```

Comment:
I suggest putting these into the commit description.

Reply:
Please remove this DescPackaging.  I don't think it's good to have this lurking in the package file.  If it is just in a commit message, it's always available.

Reply:
Agreed, it's more a CHANGELOG kind of description.